### DISCUSSION TITLE
Timer for auto disconnect

### DISCUSSION THREAD

**Discussion by brf916** [hmes98318/Music-Disc#5] [Ideas]
Could you add a feature where the bot automatically disconnects after an amount of time if nothing is in the queue. (For example if nobody puts a song in the queue for 5 minutes the bot automatically leaves the channel.

It would also be nice if you can adjust the timer (with the value in # of seconds) in config.json

**Comment by hmes98318** [comment]
Let me try.

**Comment by imwezz** [comment]
Pull request: #7
